reactor-netty-core/src/main/java/reactor/netty/Metrics.java

```diff
@@ -297,4 +308,8 @@ public static String formatSocketAddress(@Nullable SocketAddress socketAddress)
 		}
 		return null;
 	}
+
+	public static Context updateContext(Context context, Object observation) {
+		return context.hasKey(OBSERVATION_KEY) ? context : context.put(OBSERVATION_KEY, observation);
```

Comment:
So you're manually doing what the context propagation library already does, correct?

Reply:
Yeah I need only this from the context propagation library and there is no need to bring additional dependency.